src/app/screens/ReceiveInvoice/index.tsx

```diff
@@ -143,7 +143,14 @@ function ReceiveInvoice() {
     return (
       <>
         <div className="mt-4 relative p-8 bg-white rounded-lg shadow-sm ring-1 ring-black ring-opacity-5 flex justify-center items-center overflow-hidden">
-          <QRCode value={invoice.paymentRequest.toUpperCase()} />
+          <div className="relative flex flex-grid w-fit h-fit">
```

Comment:
I don't think it needs to be a grid, there's only one element. I think if you change it to `flex justify-center items-center` you might also be able to remove the transform and translate classes on the img tag too.

Reply:
Sure, I have made the changes
Thanks!